README.md

```diff
@@ -136,6 +136,12 @@ This example just streams the body of the request. The body of the response can
 
 Check out the `examples` folder how your server could look like.
 
+The first parameter (often written as `RequestInterface $request` in examples and tests) 
+of the callback function and a middleware is a `ServerRequest` object based on the 
```

Comment:
It's my understanding that consumers of this package should *not* care about the `ServerRequest` at all (this is your custom, internal implementation only). What they *should* care about is the `ServerRequestInterface`, which is part of PSR-7 and does not require much documentation here.

What do you think about this?